src/app/pages/wallets/my-wallets/pages-wallets-my-wallets.component.html

```diff
@@ -9,6 +9,6 @@
 
   <div class="my-wallets" *ngIf="myWalletsData.data">
     <app-my-wallet  *ngFor="let wallet of myWalletsData.data" [wallet]="wallet"></app-my-wallet>
-    <app-add-new-wallet></app-add-new-wallet>
+    <app-add-new-wallet (getWalletName)="getWalletName($event)"></app-add-new-wallet>
```

Comment:
Here you should have just click handler and when this component is clicked you should in `pages-wallets-my-wallets.component.ts` open the dialog for adding/updating wallet. So it should look like:
`<app-add-new-wallet (click)="openWalletModal"></app-add-new-wallet>`